### ISSUE TITLE
Items take all available width when number of items is less than the one in the responsive config object

### ISSUE THREAD

**Issue by hichemfantar** [maxmarinich/react-alice-carousel#263]
If we have 2 elements at >=1024px and the config says 3 elements.
Right now it becomes as if the first element takes up 1.5/3 and second element takes up 1.5/3.
When It should be:
First element takes up 1/3.
Second element takes up 1/3.
Third element is an empty div that takes up 1/3.
https://maxmarinich.github.io/react-alice-carousel/#basic

**Comment by maxmarinich** [comment]
Hi, @hichemfantar! Fixed in #264

**Comment by hichemfantar** [comment]
I believe you misunderstood my issue.
My issue isn't with the breakpoint, It's with the way items are rendered when the number of items is less than the number of items specified at a certain breakpoint.

**Comment by maxmarinich** [comment]
Why did the breakpoints break?
According to [code](https://github.com/maxmarinich/react-alice-carousel/blob/e5463b7df7c7f7c18bce62f6090caf39a8115e17/src/lib/utils/common.ts#L54), the case with item`s count is expected behavior:
```js
if (Number(key) <= value) {
    itemsInSlide = Math.min(responsive[key].items, itemsCount) || itemsInSlide;
}
```

**Comment by cassandrafsmith** [comment]
I'm having a similar issue. I have my response object set the same as in the basic example that you have on your example website. However, no matter what size screen I am viewing it on, only one card is displayed in the carousel.

**Comment by maxmarinich** [comment]
Fix responsive flow, add new feature in #265.
Please see https://maxmarinich.github.io/react-alice-carousel/#responsive.
